sn_cli/src/bin/main.rs

```diff
@@ -39,6 +40,22 @@ const CLIENT_KEY: &str = "clientkey";
 async fn main() -> Result<()> {
     color_eyre::install()?;
     let opt = Opt::parse();
+
+    if opt.crate_version {
+        println!("{}", env!("CARGO_PKG_VERSION"));
+        return Ok(());
+    }
+
+    if opt.protocol_version {
+        println!("{}", IDENTIFY_PROTOCOL_STR.to_string());
+        return Ok(());
+    }
+
+    if opt.package_version {
```

Comment:
Should we not restrict `package_version` to only print when not nightly? `#[cfg(not(feature = "nightly"))]`. If so, this applies to the `Opt` struct `pub package_version: bool,` field as well.